src/swc.ts

```diff
@@ -252,10 +252,27 @@ export interface TransformationOutput {
   code: string
 }
 
-export const swcVersionAtom = atom(
-  new URLSearchParams(location.search).get('version') ?? '1.2.115'
-)
+export const swcVersionAtom = atom(async () => {
+  return await getSwcVersion()
+})
 
+async function getSwcVersion() {
+  try {
+    const versionFromUrl = new URLSearchParams(location.search).get('version')
+    if (versionFromUrl) {
+      return versionFromUrl
+    }
+    const latestVersion = await getLatestVersion()
+    if (latestVersion) {
+      return latestVersion
+    } else {
+      throw new Error("can't get the latest version of @swc/wasm-web")
+    }
+  } catch (err) {
+    console.error(err)
```

Comment:
Personally I think this error can be ignored.

```suggestion
  } catch {
```